pkg/cli/start.go

```diff
@@ -1134,7 +1134,11 @@ func startShutdownAsync(
 
 			for ; ; prevRemaining = remaining {
 				var err error
-				remaining, _, err = s.Drain(drainCtx, verbose)
+				func() {
```

Comment:
Let's do this in drain.go, `drainClients`, not here.

Reply:
that's a much better idea, thanks